message_definitions/v1.0/common.xml

```diff
@@ -4766,6 +4766,16 @@
       <field type="uint8_t" name="breach_mitigation" enum="FENCE_MITIGATE">Active action to prevent fence breach</field>
     </message>
     <!-- MESSAGE IDs 180 - 229: Space for custom messages in individual projectname_messages.xml files -->
+    <message id="227" name="HYBRID_READ">
```

Comment:
I think we should rename this message to GENERATOR.  The "READ" seems inconsistent with most other messages.

Reply:
From the message above it looks like we should perhaps avoid 227 as the message id.  Maybe use one of the higher Mavlink2 message ids that are less in demand.